scm/etc/scripts/mod_forcing.py

```diff
@@ -0,0 +1,423 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+from netCDF4 import Dataset
+import os
+import f90nml
+import numpy as np
+from scipy import interpolate
+import forcing_file_common as ffc
+import shutil
+
+###############################################################################
+# Global settings                                                             #
+###############################################################################
+
+SCM_ROOT = ''
+
+# Path to default run directory (relative to scm_root)
+DEFAULT_RUN_DIR = 'scm/run'
+
+DEFAULT_CASE_INPUT_DIR = 'scm/data/processed_case_input'
+
+DEFAULT_CASE_CONFIG_DIR = 'scm/etc/case_config'
+
+###############################################################################
+# Command line arguments                                                      #
+###############################################################################
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-c', '--case',       help='name of case to modify forcing', required=True)
+parser.add_argument('--lwrad',            help='add longwave radiation tendency to forcing terms', action='store_true', default=False)
+parser.add_argument('--swrad',            help='add shortwave radiation tendency to forcing terms', action='store_true', default=False)
+parser.add_argument('--pbl',              help='add plentary boundary layer tendency to forcing terms', action='store_true', default=False)
+parser.add_argument('--deepconv',         help='add deep convective tendency to forcing terms', action='store_true', default=False)
+parser.add_argument('--shalconv',         help='add shallow convective tendency to forcing terms', action='store_true', default=False)
+parser.add_argument('--micro',            help='add microphysics tendency to forcing terms', action='store_true', default=False)
+parser.add_argument('--ogwd',             help='add orographic gravity wave drag tendency to forcing terms', action='store_true', default=False)
+parser.add_argument('--cgwd',             help='add convective gravity wave drag tendency to forcing terms', action='store_true', default=False)
+parser.add_argument('-v', '--verbose',    help='once: set logging level to debug; twice: set logging level to debug '\
+                                               'and write log to file', action='count', default=0)
+
+###############################################################################
+# Functions and subroutines                                                   #
+###############################################################################
+def setup_logging(verbose):
```

Comment:
Is this a duplicate of what is in `run_scm.py`? If so, there is some room for improvement by putting that logic in a shared Python module.

Reply:
Yes. This script is a prototype. This can be fixed when it gets made more robust/useful and is no longer a prototype.